drizzle/schema.ts

```diff
@@ -121,6 +121,7 @@ export const files1 = pgTable("files1", {
   uploaderid: integer("uploaderId").references(() => users1.id),
   projectid: integer("projectId").references(() => projects1.id),
   taskid: integer("taskId").references(() => tasks1.id),
+  specid: integer("specid"),
```

Comment:
I'd prefer not to have this here as there is now, no one source of truth. To get the specid I guess you'd have to join the files1 and the tasks1 tables and  the specid would be accessible under `file.task.specId`

Reply:
change the specId source of truth